crates/rspack_binding_options/src/options/mod.rs

```diff
@@ -114,10 +114,13 @@ impl RawOptionsApply for RawOptions {
     let node = self.node.map(|n| n.into());
     let dev_server: DevServerOptions = self.dev_server.into();
 
-    plugins.push(rspack_plugin_schemes::DataUriPlugin.boxed());
-    plugins.push(rspack_plugin_schemes::FileUriPlugin.boxed());
+    // prelude plugins
+    let mut prelude_plugins = vec![];
```

Comment:
Just curious, why need `prelude_plugins` here, did the plugins order causing some issue?

Reply:
The `LimitChunkCountPlugin` should be executed after other plugins that use the `optimize_chunks` hook.

However, I got some things mixed up. Webpack ensures this through the `stage: STAGE_ADVANCED` configuration for plugins, not through the load order of internal plugins in webpack.

I will fix it.